userComponent/src/main/kotlin/com/gravatar/app/usercomponent/data/database/model/ProfileEntity.kt

```diff
@@ -0,0 +1,98 @@
+package com.gravatar.app.usercomponent.data.database.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.gravatar.restapi.models.Profile
+import com.gravatar.restapi.models.ProfileContactInfo
+import java.net.URI
+
+@Entity(tableName = "user_profiles")
+data class ProfileEntity(
+    @PrimaryKey
+    @ColumnInfo(name = "user_id")
+    val userId: Int,
+    @ColumnInfo(name = "hash")
+    val hash: String,
+    @ColumnInfo(name = "display_name")
+    val displayName: String,
+    @ColumnInfo(name = "profile_url")
+    val profileUrl: String,
+    @ColumnInfo(name = "avatar_url")
+    val avatarUrl: String,
+    @ColumnInfo(name = "avatar_alt_text")
+    val avatarAltText: String,
+    @ColumnInfo(name = "description")
+    val description: String,
+    @ColumnInfo(name = "pronouns")
+    val pronouns: String,
+    @ColumnInfo(name = "pronunciation")
+    val pronunciation: String,
+    @ColumnInfo(name = "location")
+    val location: String,
+    @ColumnInfo(name = "job_title")
+    val jobTitle: String,
+    @ColumnInfo(name = "company")
+    val company: String,
+    @ColumnInfo(name = "first_name")
+    val firstName: String? = null,
+    @ColumnInfo(name = "last_name")
+    val lastName: String? = null,
+    @ColumnInfo(name = "contact_cell_phone")
+    val contactCellPhone: String? = null,
+    @ColumnInfo(name = "contact_email")
+    val contactEmail: String? = null,
```

Comment:
`ContactInfo` is a separate object - I've put those fields here directly and flattened it rather than creating a 2nd table. Thoughts?